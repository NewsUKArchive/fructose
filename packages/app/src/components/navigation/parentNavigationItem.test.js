/* globals describe beforeEach it expect */

import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ParentNavigationItem from "./parentNavigationItem";

configure({
  adapter: new Adapter()
});

describe("Parent Navigation Header", () => {
  let app;

  it("Renders without back icon", () => {
    app = shallow(
      <ParentNavigationItem label="label to render" onPress={() => {}} />
    );
    expect(app).toMatchSnapshot();
  });
});
