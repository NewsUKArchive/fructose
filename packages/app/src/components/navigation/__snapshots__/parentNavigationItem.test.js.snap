// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parent Navigation Header Renders without back icon 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  focusedOpacity={0.7}
  onPress={[Function]}
  style={33}
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      style={32}
    >
       
      label to render
    </Text>
    <Image
      source={
        Object {
          "testUri": "../../../packages/app/src/components/navigation/forward-icon.png",
        }
      }
      style={34}
    />
  </View>
</TouchableOpacity>
`;
